src/util/AttributeParser.js

```diff
@@ -33,6 +33,18 @@ export function isNaN(value) {
 	return Number.isNaN(value);
 }
 
+/**
+ * deep compare two values primitive or complex
+ *
+ * @param valueA
+ * @param valueB
+ * @returns {boolean}
+ */
+
+export function isNotEqual(valueA, valueB) {
```

Comment:
@quarkus I don't quite like the naming here. I think it should make it more clear that it actually is comparing deeply.
How about "deepEquals". I would also not make it inverted.